README.md

```diff
@@ -222,6 +222,18 @@ String or Array to specify address(es) for which sshd will bind. Corresponds to
 
 - *Default*: undef
 
+sshd_config_permitemptypasswords
+--------------------------------
+PermitEmptyPasswords option in sshd_config.  When password authentication is allowed, it specifies whether the server allows login to accounts with empty password strings.
```

Comment:
After the description add this to the line.

`Valid values are 'yes' and 'no'.`